super_enum_generator/pubspec.yaml

```diff
@@ -14,7 +14,8 @@ dependencies:
 
   # Specify a specific version or a version constraint
   # for "super_enum" before publishing the package.
-  super_enum: ^0.2.0
+  super_enum:
+    path: ../super_enum
```

Comment:
This dependency should be reverted before merging. I can't test it in my project with the following configuration:

```yaml
dependencies:
  super_enum:
    git:
      url: https://github.com/xsahil03x/super_enum.git
      ref: 181594b280723bf12f925b3d59c42c806ce6dce4
      path: super_enum

dev_dependencies:
  super_enum_generator:
    git:
      url: https://github.com/xsahil03x/super_enum.git
      ref: 181594b280723bf12f925b3d59c42c806ce6dce4
      path: super_enum_generator
```

Reply:
Done :+1: